src/cache.ts

```diff
@@ -47,23 +42,43 @@ export const INFINITE_QUERY_KEY = 'infinite' as const;
 
 export const createCacheUtils = <Endpoints extends RoughEndpoints>(
   client: QueryClient,
-  makeQueryKey: <Route extends keyof Endpoints & string>(
```

Comment:
This was a breaking change, right?

Reply:
@aecorredor this function is not part of the `index` export, so I consider it non-breaking